pyscriptjs/tests/unit/utils.test.ts

```diff
@@ -32,3 +33,142 @@ describe("Utils", () => {
     expect(secondElement.id).toBe("py-internal-2")
   })
 })
+
+
+describe("Test _createAlertBanner", () => {
+  document.body.innerHTML = `<div>Hello World</div>`
```

Comment:
[question] neat, I didn't know you could use `document` from jest tests.
I guess it's using a fake object which is automatically reset after each test?

Reply:
I learned this recently, all my previous attempts failed when i tried to check if something was in the document, but this seems to work and you can even do `button.click()` which is nice (I'm used to use the React testing library to do these kinds of things)

Although the body doesn't get reset after each test and seems to leak to other tests, I'll wrap this in a `afterEach` so it gets reset once we finish